sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -186,12 +188,48 @@ def managed(self):
 
 
 class KeyVaultKey(object):
-    """A key's attributes and cryptographic material"""
+    """A key's attributes and cryptographic material.
+
+    :param str key_id:
+        Key Vault's identifier for the key. Typically a URI, e.g. https://myvault.vault.azure.net/keys/my-key/version
+    :param jwk:
+        The key's cryptographic material as a JSON Web Key (https://tools.ietf.org/html/rfc7517). This may be provided
+        as a dictionary or keyword arguments. See :class:`~azure.keyvault.keys.models.JsonWebKey` for field names.
+
+    Providing cryptographic material as keyword arguments:
+
+    .. code-block:: python
+
+        from azure.keyvault.keys.models import Key
+
+        key_id = 'https://myvault.vault.azure.net/keys/my-key/my-key-version'
+        key_bytes = os.urandom(32)
+        key = Key(key_id, k=key_bytes, kty='oct', key_ops=['unwrapKey', 'wrapKey'])
+
+    Providing cryptographic material as a dictionary:
+
+    .. code-block:: python
+
+        from azure.keyvault.keys.models import Key
+
+        key_id = 'https://myvault.vault.azure.net/keys/my-key/my-key-version'
+        key_bytes = os.urandom(32)
+        jwk = {'k': key_bytes, 'kty': 'oct', 'key_ops': ['unwrapKey', 'wrapKey']}
+        key = Key(key_id, jwk=jwk)
```

Comment:
Is this supposed to be `key = KeyVaultKey(key_id, jwk=jwk)`?

Reply:
Yes, good catch 🦅